website/pages/blog/announcing-self-hosted-graphql-hive.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Announcing self-hosted GraphQL Hive
+tags: [graphql, graphql-hive]
+authors: [laurin]
+date: 2022-10-04
+description: Host your own GraphQL Hive instance.
+image: /blog-assets/announcing-self-hosted-graphql-hive/cover.png
+thumbnail: /blog-assets/announcing-self-hosted-graphql-hive/thumbnail.png
+---
+
+It has been 5 months since [we first announced GraphQL Hive](/blog/graphql-hive-preview) and it has been a wild ride since then!
+
+> GraphQL Hive is our schema registry solution for continously evolving your GraphQL schema backed by performance insights and usage statistics.
+> Whether you are working in a small company with a single GraphQL service or a big multi-team company with multiple (federated or composed) GraphQL services, GraphQL Hive is the perfect tool to help you manage your GraphQL schema.
```

Comment:
@B2o5T Don't we have quotes? Is it missing or we never had them?

![Zrzut ekranu 2022-10-4 o 14 37 50](https://user-images.githubusercontent.com/8167190/193821112-f2c4a942-a5e4-4066-8a38-e5632d6ae1af.png)

Reply:
@kamilkisiela try to clear cache, it's fine for me 
<img width="1018" alt="image" src="https://user-images.githubusercontent.com/7361780/193831621-69e10506-29f2-43ae-845c-d6e36175d019.png">